Packs/Cybereason/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Cybereason",
     "description": "Endpoint detection and response to manage and query malops, connections and processes.",
     "support": "xsoar",
```

Comment:
please change to support: "partner"
and as the pack should be adopted, please follow the adoption guidelines
https://xsoar.pan.dev/docs/partners/adopt#process